### PR TITLE
Create ProductHighlights and ProductServices

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Create the `ProductHighlights`.  See this [PR](https://github.com/vtex-apps/product-details/pull/93)

#### How should this be manually tested?
https://highlightsprod--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductHighlights/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Product Highlights
+
+## Description
+
+`ProductHighlights` is a VTEX Component that shows the general specifications of a product.
+This Component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
```
Comment (klzns) [inline react/components/ProductHighlights/README.md]:
```suggestion
:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
```

File: react/components/ProductHighlights/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - [Blocks API](#blocks-api)
+  - [CSS namespaces](#css-namespaces)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+
+## Usage
+
+You should follow the usage instruction in the main [README](https://github.com/vtex-apps/store-components/blob/master/README.md#usage).
+
+Then, add `product-highlights` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json).
```
Comment (klzns) [inline react/components/ProductHighlights/README.md]: The user have to do this?

```diff
   {
     "product-details": {
       "blocks": [
         "breadcrumb",
+        "product-highlights",
         "product-name",
         "product-images",
         "product-price",
         "product-description",
         "buy-button",
         "sku-selector",
         "shipping-simulator",
         "availability-subscriber",
         "share",
         "product-specifications"
       ]
     } 
   }
```

File: react/components/ProductHighlights/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Usage
+
+You should follow the usage instruction in the main [README](https://github.com/vtex-apps/store-components/blob/master/README.md#usage).
+
+Then, add `product-highlights` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json). 
+
+### Blocks API
+
+This component has an interface that describes which rules must be implemented by a block when you want to use the `ProductHighlights`.
```
Comment (klzns) [inline react/components/ProductHighlights/README.md]: Are there any rules? Looking at the JSON bellow it doesn't seem to.

File: react/components/ProductHighlights/index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+import HtmlParser from 'react-html-parser'
+
+
```
Comment (klzns) [inline react/components/ProductHighlights/index.js]: 🔫 

```suggestion
```

File: react/components/ProductHighlights/index.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import styles from './styles.css'
+
+/**
+ * Product highlights Component.
+ * Render the highlights specifications of a product.
+ */
+class ProductHighlights extends Component {
```
Comment (klzns) [inline react/components/ProductHighlights/index.js]: Use function instead, it compiles to a smaller file size.

File: react/components/ProductHighlights/index.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+  render() {
+    const { highlights } = this.props
+    return (
+      <ul className={styles.content}>
+        {highlights.map((item, i) => (
+          <li key={i} className={`t-heading-5`}>{HtmlParser(item.name)}: <span className={`f5`}>{HtmlParser(item.values[0])}</span> </li>
```
Comment (klzns) [inline react/components/ProductHighlights/index.js]:
```suggestion
          <li key={i} className="t-heading-5">{HtmlParser(item.name)}: <span className="f5">{HtmlParser(item.values[0])}</span> </li>
```

File: react/components/ProductServices/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+# Product Services
+
+## Description
+
+`ProductServices` is a VTEX Component that shows the services of a product.
+This Component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
```
Comment (klzns) [inline react/components/ProductServices/README.md]:
```suggestion
:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
```

File: react/components/ProductServices/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  - [Blocks API](#blocks-api)
+  - [CSS namespaces](#css-namespaces)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+
+## Usage
+
+You should follow the usage instruction in the main [README](https://github.com/vtex-apps/store-components/blob/master/README.md#usage).
+
+Then, add `product-services` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json).
```
Comment (klzns) [inline react/components/ProductServices/README.md]: Add example here

File: react/components/ProductServices/README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+## Usage
+
+You should follow the usage instruction in the main [README](https://github.com/vtex-apps/store-components/blob/master/README.md#usage).
+
+Then, add `product-services` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json). 
+
+### Blocks API
+
+This component has an interface that describes which rules must be implemented by a block when you want to use the `ProductServices`.
```
Comment (klzns) [inline react/components/ProductServices/README.md]: No rules?

File: react/components/ProductServices/index.js [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+import HtmlParser from 'react-html-parser'
+
+import { IconCheck } from 'vtex.store-icons'
+import styles from './styles.css'
+
+/**
+ * Product Services Component.
+ * Render the services of a product.
+ */
+class ProductServices extends Component {
```
Comment (klzns) [inline react/components/ProductServices/index.js]: Use function instead

File: react/components/ProductServices/index.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+  render() {
+    const { services, iconSize } = this.props
+    return (
+      <div className={styles.content}>
+        {services
+          .filter(item => item.values[0].toLowerCase() === 'enable')
+          .map((item, i) => (
+            <div key={i} className={'flex-row dib ma2 ph2'}>
```
Comment (klzns) [inline react/components/ProductServices/index.js]: Use string literal instead

File: react/components/ProductServices/index.js [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+              <span className={`c-success`}>
+                <IconCheck type="line" size={iconSize} />
+              </span>
+              <span className={`f5`}>{HtmlParser(item.name)}</span>
```
Comment (klzns) [inline react/components/ProductServices/index.js]: same

File: react/components/ProductServices/index.js [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+class ProductServices extends Component {
+  render() {
+    const { services, iconSize } = this.props
+    return (
+      <div className={styles.content}>
+        {services
+          .filter(item => item.values[0].toLowerCase() === 'enable')
+          .map((item, i) => (
+            <div key={i} className={'flex-row dib ma2 ph2'}>
+              <span className={`c-success`}>
```
Comment (klzns) [inline react/components/ProductServices/index.js]: same

File: react/components/ProductHighlights/index.js (hunk 6 above)
Comment (arthurepc) [inline react/components/ProductHighlights/index.js]: Also, have you thought whether using the index as a key is a good fit here or not? [This article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) is a good source of knowledge

File: react/components/ProductServices/index.js (hunk 11 above)
Comment (arthurepc) [inline react/components/ProductServices/index.js]: Same about using index as key

**Pull request merged**